README.md

```diff
@@ -1,73 +1,68 @@
 # Nova quickstart
 
-This is a simple example of a nov base Discord bot inspired by the discord.js structure.
+This document illustrates the process of creating a Discord Bot using the 
+Nova platform.
+The Bot interacts with Nova using a library following the popular 
+[Discord.js](https://discord.js.org) structure.
```

Comment:
Do not refer to discord.js at the top of the readme. We are base on discord.js libraries (types, rest etc) but the gateway is custom.
You can add a reference to discord.js in the links or in the library used section (need to be created)